src/cmd/fix.rs

```diff
@@ -63,30 +82,72 @@ pub enum Error {
     /// Generating the debug bundle.
     #[error("generate debug bundle")]
     GenerateDebugBundle,
+
+    /// Downloading cli
+    #[error("download fossa cli")]
+    DownloadFossaCli {
+        /// A message explaining how to fix this error
+        msg: String,
+        /// The error returned from downloading the cli
+        error: String,
+    },
+
+    /// Cloning Reference
+    #[error("clone reference")]
+    CloneReference {
+        /// The reference that failed to be cloned
+        reference: Reference,
+        /// The error returned from cloning the reference
+        error: String,
+        /// A message explaining how to fix this error
+        msg: String,
+    },
+}
+
+#[cfg(target_family = "windows")]
+fn cli_command() -> &'static str {
+    "PATH ; fossa.exe analyze -o"
+}
```

Comment:
I don't think this is how env vars are set for the current session in powershell. Please validate- if I'm wrong I'm happy to accept, but otherwise please fix it.

Also, I think I never mentioned before that when we say "windows" we mean "windows _in powershell_". Sorry about that.